db/re/item_db.conf

```
@@ -31808,10 +31808,16 @@ item_db: (
 	Name: "Rune Boots"
 	Type: "IT_ARMOR"
 	Buy: 20
-	Weight: 100
+	Weight: 10
 	Def: 5
 	Slots: 1
 	Loc: "EQP_SHOES"
+	Script: <"
+		if(readparam(bStr) >= 90 && readparam(bStr) <= 119) bonus bAtk, 10;
+		if(readparam(bStr) >= 120) bonus bAtk, 20;
+		if(readparam(bInt) >= 90 && readparam(bInt) <= 119) bonus bMatkRate, 3;
+		if(readparam(bInt) >= 120) bonus bMatkRate, 5;
```

```suggestion
		if (readparam(bStr) >= 90) bonus bAtk, 10;
		if (readparam(bStr) >= 120) bonus bAtk, 20;
		if (readparam(bInt) >= 90) bonus bMatkRate, 3;
		if (readparam(bInt) >= 120) bonus bMatkRate, 5;
```
effect are stackable when stats are higher...

---

oh yeah. i forgot that part.